bitnami/consul/templates/statefulset.yaml

```diff
@@ -18,14 +18,11 @@ spec:
   template:
     metadata:
       labels: {{- include "consul.labels" . | nindent 8 }}
-      {{- if or .Values.podAnnotations (and .Values.metrics.enabled .Values.metrics.podAnnotations) }}
+      {{- if .Values.podAnnotations }}
```

Comment:
Finally, I would leave this as it is and we can cover both use cases. What do you think?

Reply:
ok. You are right, I will update it later.